include/nbl/builtin/hlsl/blit/blit/descriptors.hlsl

```diff
@@ -0,0 +1,84 @@
+
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_HLSL_BLIT_NORMALIZATION_DESCRIPTORS_INCLUDED_
+#define _NBL_HLSL_BLIT_NORMALIZATION_DESCRIPTORS_INCLUDED_
+
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace blit
+{
+namespace normalization
+{
+
```

Comment:
this header is not needed because we will use Lambda-like structs and HLSL templates

Reply:
small example
```cpp
struct PassedCounterAlphaHistoramAccessor
{
    void increment(const uint bucketIx);
};
```